File: b532.字串處理/main.py
from math import *
row=int(input())

for i in range(row):
    check=False
    my_string=input()
    front=""
    back=""
    opper=""
    for letter in my_string:
        if letter=="+" or letter=="-" or letter=="*" or letter=="/" or letter=="%":
            opper+=letter
            check=True
        if letter.isdigit():
            if check:
                back+=letter
            else:
                front+=letter
    if front[0]=="0":
        front=front[1:]
    if back[0]=="0":
        back=back[1:]
    ans=front+opper+back
    print(floor(eval(ans)))

File: b532.字串處理/main.cpp
#include<iostream>
#include<string>
using namespace std;
int math(int a, char b, int c)
{
	switch (b)
	{
	case '+':
		return a + c;
	case '-':
		return a - c;
	case '*':
		return a * c;
	case '/':
		return a / c;
	case'%':
		return a % c;
	}
}
int main(void)
{
	int n ;
	cin >> n;
	for (int i = 0; i < n; i++)
	{
		bool check = false;
		string a="", front="", back="", opper="";
		cin >> a;
		for (char k : a)
		{
			if (k == '+' || k == '-' || k == '*' || k == '/' || k == '%')
			{
				opper.push_back(k);
				check = true;
			}
			else if (isdigit(k))
			{
				if (check)
					back.push_back(k);
				else
					front.push_back(k);
			}
			
		}
		//cout << stoi(front) << opper[0] << stoi(back);
		cout<<math(stoi(front),opper[0],stoi(back))<<endl;
	}
}